ddapm_test_agent/agent.py

```diff
@@ -51,6 +51,9 @@
 log = logging.getLogger(__name__)
 
 
+AGENT_PROXY_HOST = os.environ.get("PROXY_HOST", "127.0.0.1")
```

Comment:
this should be set like the rest of the arguments/settings:

https://github.com/DataDog/dd-apm-test-agent/blob/232a2c6b92a2b032357741895a49c92c92dce860/ddapm_test_agent/agent.py#L725-L730

Reply:
removed this argument, have no need for it for PHP